package.json

```diff
@@ -21,7 +21,7 @@
     "expect": "^1.6.0",
     "isparta": "^3.0.3",
     "mocha": "^2.2.5",
-    "react": ">=0.13.2 || ^0.14.0-beta1"
+    "react": ">=0.13.2 || ^15.0.2"
```

Comment:
Actually I have noticed something wrong with this, it should include all of 15 branch and I am not sure about the piping. Could we not just do >=0.13.2 < 16.0.0